src/pages/ProgramAdminPage/ProgramCoverForm.tsx

```diff
@@ -129,11 +152,13 @@ const ProgramCoverForm: React.VFC<{
         coverDefaultUrl: uploadCoverUrl,
         coverMobileUrl: uploadMobileUrl,
         coverThumbnailUrl: uploadThumbnailUrl,
+        coverType: coverMedia ? validateFile(coverMedia.name) : 'image',
+        mobileCoverType: coverMobileMedia ? validateFile(coverMobileMedia.name) : 'image',
```

Comment:
Using `coverMedia.type`.
Please try `console.log(coverMedia)` to see data  when uploading the file.